apps/crawler/src/id_crawler.mjs

```diff
@@ -0,0 +1,258 @@
+import { chromium } from "playwright";
+import * as cheerio from "cheerio";
+import fs from "fs";
+import path from "path";
+import { fileURLToPath } from "url";
+import axios from "axios";
+
+// ES Module compatible __dirname
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const BASE_URL = "https://publicity.businessportal.gr";
+const PAGE_LOAD_TIMEOUT_PUPPETEER = 60000;
```

Comment:
We no longer use Puppeteer, we can rename this variable to something more generic.
```suggestion
const PAGE_LOAD_TIMEOUT_IN_MILLISECONDS = 60000;
```